ui/desktop/electron-app/src/index.js

```diff
@@ -62,7 +62,6 @@ const createWindow = (partition, closeWindowCB) => {
       webSecurity: true,
       contextIsolation: true,
       nodeIntegration: false,
-      enableRemoteModule: false,
```

Comment:
Delete this line because it has been deprecated, and being `false` is the default behaviour. [More info about here](https://www.electronjs.org/docs/latest/breaking-changes#removed-remote-module)